src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -227,6 +229,7 @@ internal TabItem AddTabItem(IViewExtension viewExtension, ContentControl content
                 TabItem tab = new TabItem();
                 tab.Header = viewExtension.Name;
                 tab.Tag = viewExtension.GetType();
+                tab.Uid = viewExtension.UniqueId;
```

Comment:
are we going to stick to name or use this?

Reply:
@mjkkirschner Please use ID if we can, do we need to update AC?